addons/l10n_cl/views/report.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account_invoices_without_payment" model="ir.actions.report">
+        <field name="name">PDF with Copy</field>
+        <field name="model">account.move</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">l10n_cl.yielding_invoice</field>
```

Comment:
name of the template could be a bit better like report_invoice_document_copy or something like this